proto/minder/v1/minder.proto

```diff
@@ -114,7 +114,7 @@ message SignatureVerification {
     optional string cert_identity = 4;
     optional string cert_issuer = 5;
     optional string rekor_log_id = 6;
-    optional int32 rekor_log_index = 7;
+    optional double rekor_log_index = 7;
```

Comment:
I'm not sure this is the right data type to use for a rekor log index. @lukehinds can you double check this? In the rekor OpenAP spec [1] I see instances of `integer` while the golang API uses `int64` [2]. Finally, the Rekor protobuf seems to point to an int64 as a log index [3]

[1] https://github.com/sigstore/rekor/blob/main/openapi.yaml#L184-L189
[2] https://github.com/sigstore/rekor/blob/main/pkg/api/api.go#L63
[3] https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_rekor.proto#L94

Reply:
I think int64 is the right choice.